common/src/main/java/com/wynntils/models/gear/GearInfoRegistry.java

```diff
@@ -147,45 +123,21 @@ private void loadGearRegistry() {
         });
     }
 
-    private static final class GearMajorIdDeserializer implements JsonDeserializer<GearMajorId> {
-        @Override
-        public GearMajorId deserialize(JsonElement jsonElement, Type jsonType, JsonDeserializationContext context)
-                throws JsonParseException {
-            JsonObject json = jsonElement.getAsJsonObject();
-
-            return new GearMajorId(
-                    JsonUtils.getNullableJsonString(json, "id"),
-                    JsonUtils.getNullableJsonString(json, "name"),
-                    StyledText.fromString(JsonUtils.getNullableJsonString(json, "lore")));
-        }
-    }
-
     private static final class GearInfoDeserializer implements JsonDeserializer<GearInfo> {
-        private final List<GearMajorId> allMajorIds;
-
-        private GearInfoDeserializer(List<GearMajorId> allMajorIds) {
-            this.allMajorIds = allMajorIds;
-        }
-
         @Override
         public GearInfo deserialize(JsonElement jsonElement, Type jsonType, JsonDeserializationContext context)
                 throws JsonParseException {
             JsonObject json = jsonElement.getAsJsonObject();
 
-            // Wynncraft API has two fields: name and displayName. The former is the "api name", and is
-            // always present, the latter is only present if it differs from the api name.
-            // We want to store this the other way around: We always want a displayName (as the "name"),
-            // but if it the apiName is different, we want to store it separately
+            // Wynncraft API name field includes "unnormalized" characters like ֎, we want to remove them
             String primaryName = json.get("name").getAsString();
-            String secondaryName = JsonUtils.getNullableJsonString(json, "displayName");
```

Comment:
Did they remove the distinction of name/displayName? How do they handle the conflict where there were some item that originally was called "Foo" but now is called "Bar" in the user visible code (but still "Foo" in the gear view screen thingie), and there is a new item that is called "Foo" in user visible code.

Sorry for the handwaving, I have forgotten the details. But I know there were some problematic items.

Reply:
Yes, so basically the case is the same as it used to, except they fixed all mis-matched names. But items like `Cancer֎` still exist, since the old Cancer item is not called `Necrosis`. But The API mostly correctly returns all names, and those that are not normalised are handled below.

Reply:
I don't still get it. Using your examples, I thought this was the case: (using your example)

Item 1: displayName: `Necrosis`, internalRollName: `Cancer`
item 2: displayName: `Cancer`, internalRollName:`Cancer֎`

I fail to understand how they can tell us that an item should be shown to the user as `Necrosis`, but called `Cancer` in the internal roll, if they only have a single name provided per item.

Reply:
Yes, it's not right. I brought this up with the API devs, hopefully we can get the old name/displayName system back.

Reply:
We've managed to get it back into the API, this should be fixed now.